pcl_detector/src/pcl_detector_ros_node.cpp

```diff
@@ -0,0 +1,11 @@
+#include <pcl_detector/pcl_detector_ros.hpp>
+
+
+int main(int argc, char * argv[])
+{
+  const rclcpp::NodeOptions options;
```

Comment:
Are these options used? Either make the node respect the options (recommended) or remove this line